### ISSUE TITLE
Check size.speed in solution file names match actual run result

### ISSUE THREAD

**Issue by atesgoral** [atesgoral/hrm-solutions#108]

**Comment by atesgoral** [comment]
Checking size. Can't check speed until speed measurements match the game's.

**Comment by Mygod** [comment]
@meh2481 is working on this (I think): http://goofans.com/forum/more-2d-boys/human-resource-machine/14101#comment-49236
Meanwhile you can try out his hrmTools and see if you can figure out what that bin file says.

**Comment by atesgoral** [comment]
@Mygod Yeah, thanks to @meh2481 we got a hold of the extracted data file, but the format of level data is not immediately evident. Someone will have to do some reverse-engineering...

**Comment by meh2481** [comment]
I'm not currently working on it. If reverse-engineering file formats isn't your forte, then contacting Allan would be the best way to go. https://twitter.com/allan_blomquist or allanblomquist@experimentalgameplay.com

**Comment by atesgoral** [comment]
@meh2481 I just did. :luck:

**Comment by atesgoral** [comment]
As a follow up, Allan actually responded very quickly with a lengthy description that will help me improve the inbox generation.

**Comment by sixlettervariables** [comment]
Looking forward to reading about it!
